workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Renderer/Renderer.tsx

```diff
@@ -264,9 +270,16 @@ const Renderer: React.FC<RendererProps> = ({
 
 								return (
 									<Form.Checkbox
-										checked={form[name]?.includes(
-											optionValue
-										)}
+										checked={
+											(Array.isArray(form[name]) &&
+												form[name]
+													.map(
+														({value}: Options) =>
+															value
+													)
+													.includes(optionValue)) ||
+											form[name].includes(optionValue)
```

Comment:
```suggestion
											(Array.isArray(form[name]) &&
												form[name]
													.some(
														(option: Options | string) =>
                                                            typeof option === 'string'
                                                                ? option === optionValue
                                                                : option.value === optionValue
													)
```

I believe this simplifies this statement. Could you give it a check?

Reply:
Thanks for the suggestion :fire: